app/Wiki.php

```diff
@@ -87,6 +87,11 @@ public function wikiLifecycleEvents(): \Illuminate\Database\Eloquent\Relations\H
         return $this->hasOne(WikiLifecycleEvents::class);
     }
 
+    public function wikibaseNotificationSentRecord(): \Illuminate\Database\Eloquent\Relations\HasMany
```

Comment:
Not sure if there is an existing convention around it, but should this method be named `wikibaseNotificationSentRecords` as it's `HasMany`? The docs do this https://laravel.com/docs/10.x/eloquent-relationships#one-to-many